pmd-core/src/main/java/net/sourceforge/pmd/cache/FileAnalysisCache.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+package net.sourceforge.pmd.cache;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Map;
+
+import net.sourceforge.pmd.PMD;
+
+/**
+ * An analysis cache backed by a regular file.
+ */
+public class FileAnalysisCache extends AbstractAnalysisCache {
```

Comment:
Nothing in particular, that' why I did it this way. But still wanted someone else's opinion.